cardano-node/src/Cardano/Node/Run.hs

```diff
@@ -105,6 +106,23 @@ runNode
   :: PartialNodeConfiguration
   -> IO ()
 runNode cmdPc = do
+    -- Workaround to ensure that the main thread throws an async exception on
+    -- receiving a SIGTERM signal.
+#ifdef UNIX
+    -- Similar implementation to the RTS's handling of SIGINT (see GHC's
+    -- TopHandler.hs)
```

Comment:
I think it's worth providing a permalink to `TopHandler.hs` so it's easy to find that resource.

Reply:
Will do in https://github.com/input-output-hk/cardano-node/pull/3641